.storybook/themes/commerce.scss

```diff
@@ -3,43 +3,589 @@
+						background-color: transparent,
+						border-color: var(--global-neutral-10),
+						color: var(--global-neutral-10),
+					),
+					button-primary-reverse-hover: (
+						border-color: var(--global-neutral-7),
+						color: var(--global-neutral-7),
+					),
+					button-secondary: (
+						background-color: var(--global-white),
+						color: var(--global-neutral-10),
+					),
+					button-secondary-hover: (
+						background-color: var(--global-neutral-7),
+						color: var(--global-white),
+					),
+					button-secondary-reverse: (
+						border-color: var(--global-white),
+						background-color: transparent,
+						color: var(--global-white),
+					),
+					button-secondary-reverse-hover: (
+						border-color: var(--global-neutral-7),
+						background-color: transparent,
+						color: var(--global-neutral-7),
+					),
+					// DESIGN ASSUMPTION
+					// "Best match" assumptions were made w/ color and background
+					// color for the "default" this variant, based on both design in
+					// Zeplin as well as design trends in Figma. Please make any corrections.
+					button-default:
+						(
+							color: var(--global-neutral-10),
+							background-color: var(--global-white),
+						),
+					button-default-hover: (
+						color: var(--global-neutral-10),
+						background-color: var(--global-neutral-1),
+					),
+					// DESIGN ASSUMPTION: Date does not require component styling.
+					// DESIGN ASSUMPTION: Grid does not require component styling.
+					carousel:
+						(
+							--slides: 1,
+						),
+					carousel-actions: (
+						display: none,
+						padding-left: var(--global-spacing-3),
+						padding-right: var(--global-spacing-3),
+						//
+						// The width should place the action buttons near the edge of the carousel responsively
+						//
+						// width is derived from adding
+						//   --layout-content-max-width
+						//   2x button widths (--global-font-size-9)
+						//   2x button margins (--global-font-size-5)
+						// then dividing those by
+						//   --layout-max-width
+						// using the formula: (48 + 32 + 1216 + 32 + 48) / 1440
+						//
+						width:
+							calc(calc(var(--layout-content-max-width) + 160) / var(--layout-max-width) * 100%),
+					),
+					carousel-button: (
+						box-shadow: var(--global-box-shadow-1),
+						border-radius: var(--global-spacing-1),
+						background-color: var(--global-white),
+						height: var(--global-spacing-7),
+						width: var(--global-spacing-7),
+					),
+					carousel-track: (
+						gap: var(--global-spacing-2),
+						width: var(--layout-content-scale-width),
+					),
+					heading: (
+						font-weight: var(--global-font-weight-7),
+					),
+					icon: (
+						fill: var(--global-neutral-8),
+						height: var(--global-spacing-4),
+						width: var(--global-spacing-4),
+					),
+					// DESIGN QUESTION:
+					// There are multiple paddings for input in Figma.
+					// Is this the correct one for a standard, default text input?
```

Comment:
I see below you are making use of `margin-top` and `margin-bottom` values. We should avoid using `margin`s to do spacing, and use `flex` with gap to ensure consistent spacing. We can then see the spacing on the input as a whole and any children would get the same spacing applied

```
input: (
    align-items: flex-start,
	display: flex,
	flex-direction: column,
	gap: var(--global-spacing-1),
),
```

Reply:
has this been addressed @ecarlisle ?

Reply:
yes, this has ^^